packages/lib-utils/src/index.ts

```diff
@@ -1,2 +1,11 @@
 export { UtilsConfig, setUtilsConfig, getUtilsConfig } from './config';
 export { commonFetch, commonFetchText, commonFetchJSON } from './utils/common-fetch';
+export {
+  k8sGet,
+  k8sCreate,
+  k8sUpdate,
+  k8sPatch,
+  k8sDelete,
+  k8sList,
+  k8sListItems,
```

Comment:
We went with `k8s{crud}Resource` for these calls... I wonder if we want to keep that name 🤔 

I'm indifferent, not sure it matters... but it'll likely need to be aliased in OpenShift Console. Also HAC-Dev believes the calls will have a suffixed `...Resource` so the adoption will take an extra hit when they have the NPM package included.

Reply:
I'm also indifferent here, I can add the `Resource` suffix if it's more meaningful.

Reply:
Let's go with the `Resource` suffix, everything is easier that way.